evansmusomi/burger-builder
# Refactors App for Optimization

*What this does*
- Shares the check form validity logic used in checkout and auth process
- Disables viewing of redux state via dev tools on production environments
- Lazy loads some routes: orders, auth, checkout

*How to verify*
- A developer should be able to confirm that the same function is used to check the validity of auth and checkout forms
- A developer should be able to confirm that redux dev tools are only enabled in development mode
- A developer should be able to confirm that additional js bundles are loaded when a user first navigates to the lazy loaded routes
